plugins/org.eclipse.fordiac.ide.export.forte_ng/src/org/eclipse/fordiac/ide/export/forte_ng/simple/SimpleFBImplTemplate.xtend

```diff
@@ -50,5 +48,30 @@ class SimpleFBImplTemplate extends BaseFBImplTemplate<SimpleFBType> {
 		  «ENDFOR»
```

Comment:
Please remove the sending of all output events as per the previous semantics.

Reply:
I had overlooked that part